driver-core/src/test/resources/unified-test-format/change-streams/change-streams.yml

```diff
@@ -0,0 +1,74 @@
+description: "change-streams"
```

Comment:
In java driver we've chosen not to copy the YAML files, but rather only the JSON files.  Can you remove it?